### ISSUE TITLE
Update JSON helper

### ISSUE THREAD

**Issue by alexlemaire** [kaskadi/kaskadi-cli#14]
@Holger-Will while working on `kaskadi-template-api` revamping and manipulating JSON I realized something for `kaskadi-cli`...

We use [this](https://github.com/kaskadi/kaskadi-cli/blob/master/functions/init/helpers/update-package-json.js) function inside of our `init` functions in order to update `package.json`.

But for everything else we use [that](https://github.com/kaskadi/kaskadi-cli/blob/master/functions/init/helpers/replace-name-in-file.js) (including `serverless.yml` files for me in all templates for AWS services).

I feel like when using `update-package-json.js` we are calling multiple functions to in the end do something we could do in 1 go with `replace-name-in-file.js`.
Basically if we call `replace-name-in-file.js` we would read `package.json` as string, do the replacement and write it back directly.
With `update-package-json.js` we are reading, then parsing, then recursively replacing in the object to then stringify it back and write it in the file. I feel like that's a lot of operations to achieve the same result.

I think we could call `replace-name-in-file.js` for everything which needs a replacement in file of some placeholders. And refactor `update-package-json.js` to target and update some specific fields in a JSON file.

What do you think?

**Comment by alexlemaire** [comment]
Refactored init logic (+ code cleanup & test suite update). I kind of implemented the modification suggested above: simply removed `update-package-json.js` + its dependencies and now handling the whole replacement logic via `replace-name-in-file.js`.

The test suite now runs 3 times faster (around 300ms instead of 1s). There are also 5 less files to maintain.

In case you have a look at the changes: sorry I didn't work with PR because I was initially going in there for a modification related to our API template. But then I realized that I could do that at the same time since I was there...

Closing for now!
